dnnlib/tflib/tfutil.py

```diff
@@ -125,7 +125,7 @@ def assert_tf_initialized():
         raise RuntimeError("No default TensorFlow session found. Please call dnnlib.tflib.init_tf().")
 
 
-def create_session(config_dict: dict = None, force_as_default: bool = False) -> tf.Session:
+def create_session(config_dict: dict = None, force_as_default: bool = False) -> tf.compat.v1.Session:
     """Create tf.Session based on config dict."""
     # Setup TensorFlow config proto.
     cfg = _sanitize_tf_config(config_dict)
```

Comment:
In line 132, I had to also update

``` 
config_proto =  tf.compat.v1.ConfigProto()
```

Reply:
hi, do you have full working code for this repo but using tf 2.x?

Reply:
@atreeleaf

Reply:
@atreeleaf I'll review the codebase again and replace any other depreciated functions. Thanks for bringing this to my attention

Reply:
@atreeleaf @PsVenom 
Hi all! Is there a running Tensorflow2 compatible StyleGAN2 Colab notebook? Thank you!